sdk/python/tests/integration/online_store/test_universal_online.py

```diff
@@ -607,12 +608,12 @@ def test_online_retrieval(environment, universal_data_sources, full_feature_name
 
     entity_rows = [
         {
-            "driver": driver,
-            "customer_id": customer,
+            "driver": _driver,
```

Comment:
why is this change being made? what is the difference between driver and origin id here?

Reply:
`driver` was being shadowed by an imported function so I changed the loop variable name.